tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -714,4 +714,61 @@ public function userShouldBeAbleToSendShareInvitationWithAllAllowedPermissionRol
 		}
 		Assert::assertTrue($areAllSendInvitationSuccessFullForAllowedRoles, $shareInvitationRequestResult);
 	}
+
+	/**
+	 * @When /^user "([^"]*)" (?:tries to send|sends) the following share invitation using root endpoint of the Graph API:$/
+	 *
+	 * @param string $user
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 * @throws \GuzzleHttp\Exception\GuzzleException
+	 */
+	public function userSendsTheFollowingShareInvitationForSpaceUsingRootEndPointTheGraphApi(string $user, TableNode $table):void {
+		$shareeIds = [];
+		$shareTypes = [];
+		$rows = $table->getRowsHash();
+		if ($rows['space'] === 'Personal' || $rows['space'] === 'Shares') {
+			$space = $this->spacesContext->getSpaceByName($user, $rows['space']);
+		} else {
+			$space = $this->spacesContext->getCreatedSpace($rows['space']);
+		}
+		$spaceId = $space['id'];
+
+		if (\array_key_exists('shareeId', $rows)) {
+			$shareeIds[] = $rows['shareeId'];
+			$shareTypes[] = $rows['shareType'];
+		} else {
+			$sharees = array_map('trim', explode(',', $rows['sharee']));
+			$shareTypes = array_map('trim', explode(',', $rows['shareType']));
+
+			foreach ($sharees as $sharee) {
+				// for non-exiting group or user, generate random id
+				$shareeIds[] = $this->featureContext->getAttributeOfCreatedUser($sharee, 'id')
+					?: $this->featureContext->getAttributeOfCreatedGroup($sharee, 'id');
+			}
+		}
+
+		$permissionsRole = $rows['permissionsRole'] ?? null;
+		$permissionsAction = $rows['permissionsAction'] ?? null;
+		$expireDate = $rows["expireDate"] ?? null;
+
+		$response = GraphHelper::sendSharingInvitationForDriveRoot(
+			$this->featureContext->getBaseUrl(),
+			$this->featureContext->getStepLineRef(),
+			$user,
+			$this->featureContext->getPasswordForUser($user),
+			$spaceId,
+			$shareeIds,
+			$shareTypes,
+			$permissionsRole,
+			$permissionsAction,
+			$expireDate
+		);
+
+		if ($response->getStatusCode() === 200) {
+			$this->featureContext->shareNgAddToCreatedUserGroupShares($response);
```

Comment:
here is question (not related your current PR). we invite member and save last created share. 

if we want to change or delete invite we get permissionsId (like: `u:534bb038-6f9d-4093-946f-133be61fa4e7`) using `shareNgGetLastCreatedUserGroupShare()`

but what should I do if: 
- I create multiple member invitation (invite `user1`, invite `user2`. How get share of `user1`?) 
- I want to delete myself from space (we haven't last `LastCreatedUserGroupShare` in this case)


I see that `permissionsId` in case `InvitationForDriveRoot` is `u: userUUD` 
maybe we shouldn't here save `LastCreatedUserGroupShare` instead of that use users/group UUID?

Reply:
>I create multiple member invitation (invite user1, invite user2. How get share of user1?)

For this, we can implement a method like `shareNgGetCreatedUserGroupShare(share, sharer)` to get the needed share from the created shares pool. This way prevent doing requests to the server to get the user/group id

>I want to delete myself from space (we haven't last LastCreatedUserGroupShare in this case)

This is exceptional case, so we can just request the user id in this case only.

Or, we can utilize `createdGroups` and `createdUsers` to get the required ids and do not use `shareNgAddToCreatedUserGroupShares` for drive shares

Reply:
>we can utilize createdGroups and createdUsers to get the required ids and do not use shareNgAddToCreatedUserGroupShares for drive shares

@ScharfViktor this looks good to me

Reply:
for me too